drivers/i2c/i2c_ite_enhance.c

```diff
@@ -523,11 +520,13 @@ static int i2c_enhance_transfer(const struct device *dev, struct i2c_msg *msgs,
 
 		if (msgs->flags & I2C_MSG_START) {
 			data->i2ccs = I2C_CH_NORMAL;
-			/* enable i2c interrupt */
-			irq_enable(config->i2c_irq_base);
 		}
+
 		/* Start transaction */
-		i2c_transaction(dev);
+		if (i2c_transaction(dev)) {
```

Comment:
Please add a comment to the `i2c_transaction()` and the `enhanced_i2c_tran_read/write()` documenting the return value.

Reply:
Done.